.gitignore

```diff
@@ -15,4 +15,5 @@
 # vendor/
 .idea/
 db/
-testscenario/
\ No newline at end of file
+testscenario/
+config/application.yml
```

Comment:
This is not a good idea. If we make changes to the file (add new properties, update, etc.), it will be ignored.